packages/react/src/stories/Card/DetailCard.stories.tsx

```diff
@@ -17,6 +17,10 @@ const CardMeta: Meta<typeof DetailCard> = {
       options: ["wide", "narrow", "fluid"],
       control: { type: "radio" },
     },
+    titleElement: {
+      options: ["h1", "h2", "h3", "h4", "h5"],
```

Comment:
@Shashika6 for all cards, this should include "p" or else the user won't be able to return to the default.